github-upload-secrets

```diff
@@ -94,17 +94,22 @@ def upload_secret(api, path, name, value, pubkey, opts):
 
 
 def upload_deploy_key(api, repo, title, content, opts):
-    keys_path = f"/repos/{repo}/keys"
-
-    # delete all existing keys
-    keys = api.get(keys_path)
-    for key in keys:
-        key_id = key['id']
-        key_path = f'{keys_path}/{key_id}'
-        if opts.dry_run or opts.verbose:
-            print(opts.dry_run and "Would delete" or "Deleting", key_path)
-        if not opts.dry_run:
-            api.delete(key_path)
+    keys_path = f"/repos/{repo.lstrip('+')}/keys"
+
+    if repo.startswith('+'):
+        # add this key to the existing keys
+        pass
```

Comment:
Bit weird to use `pass` instead of negating the condition, but ok..

Reply:
I agree, but I really wanted the comment here.